cartography/rules/spec/output.py

```diff
@@ -0,0 +1,30 @@
+# Execution result classes
+from dataclasses import dataclass
+from typing import Any
+
+
+@dataclass
+class FactResult:
+    fact_id: str
+    fact_name: str
+    fact_description: str
+    fact_provider: str
+    requirement_id: str
+    finding_count: int = 0
+    findings: list[dict[str, Any]] | None = None
+    requirement_url: str | None = None
+
```

Comment:
Should we also have a Requirements Result? This way in the output we can cleanly go from Framework -> Requirement -> Fact and if a Requirement within a Framework is all green we can just minimize those results and put a big green check mark or something. Keeps the data flow consistent throughout

Reply:
Would require some slight changes to the runners I suppose